trio/_tools/mypy_annotate.py

```diff
@@ -1,9 +1,19 @@
 """Translates Mypy's output into GitHub's error/warning annotation syntax.
 
 See: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions
+
+This first is run with Mypy's output piped in, to collect messages in
+mypy_annotate.dat. After all platforms run, we run this again with
+--export, which prints the messages but with cross-platform failures
```

Comment:
The code doesn't use this `--export` flag.

Reply:
Ahh yeah, I'm just using the absence of `--platform` instead.